hueplusplus/test/mocks/mock_linHttpHandler.h

```diff
@@ -0,0 +1,40 @@
+/**
+	\file mock_linHttpHandler.h
+	Copyright Notice\n
+	Copyright (C) 2017  Jan Rogall		- developer\n
+	Copyright (C) 2017  Moritz Wirger	- developer\n
+
+	This program is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation; either version 3 of the License, or
+	(at your option) any later version.
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+	You should have received a copy of the GNU General Public License
+	along with this program; if not, write to the Free Software Foundation,
+	Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301  USA
+**/
+
+#ifndef _MOCK_LIN_HTTPHANDLER_H
+#define _MOCK_LIN_HTTPHANDLER_H
+
+#include <string>
+#include <vector>
+
+#include <gmock/gmock.h>
+
+#include "../hueplusplus/include/linHttpHandler.h"
+#include "../hueplusplus/include/json/json.h"
+
+//! Mock Class
+class MockLinHttpHandler : public linHttpHandler

```

Comment:
This mock only exists for linHttpHandler. The tests fail to build on Windows.
MockLinHttpHandler is not used anywhere in the tests, it could just be removed.

Reply:
still not fixed